### PR TITLE
Rename cucoslib to f8a_worker

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -77,5 +77,5 @@ target/
 .idea/*
 
 # Misc
-cucos-license-check-unreleased.rpm
+f8a-license-check-unreleased.rpm
```
Comment (jpopelka) [inline .gitignore]: I'd remove this completely.

File: Dockerfile.tests [hunk 2]
```diff
@@ -3,16 +3,16 @@ MAINTAINER Tomas Tomecek <ttomecek@redhat.com>
...
+COPY . /f8a_worker
+WORKDIR /f8a_worker
 RUN cd tests && pip3 install -r ./requirements.txt
 
-# Install cucos-license-check prerelease if present in lib dir
-RUN ls cucos-license-check-unreleased.rpm && \
-    dnf install -y cucos-license-check-unreleased.rpm; \
+# Install f8a-license-check prerelease if present in lib dir
+RUN ls f8a-license-check-unreleased.rpm && \
+    dnf install -y f8a-license-check-unreleased.rpm; \
```
Comment (jpopelka) [inline Dockerfile.tests]: Just remove it, we don't need this anymore.

File: README.md [hunk 3]
```diff
@@ -61,28 +59,28 @@ spurious test failures.
...
 
 # Testing against not-yet-released worker dependencies
 
 The `Dockerfile.tests` file is set up to install any
-`cucos-license-check-unreleased.rpm` file stored locally in this directory.
+`f8a-license-check-unreleased.rpm` file stored locally in this directory.
 In combination with the `license-check-worker/make_rpm.sh` script in the
 data-mining-tools repo, this feature can be used to run the worker tests
-against a version of `cucos-license-check` that is not yet released.
+against a version of `f8a-license-check` that is not yet released.
```
Comment (jpopelka) [inline README.md]: This goes back to the cucos days. Remove the `Testing against not-yet-released worker dependencies` completely.

**Comment by fridex** [comment]
A lot changed, creating new PR in https://github.com/fabric8-analytics/fabric8-analytics-worker/pull/76

**Pull request closed without merging**